chromium_src/components/search_engines/template_url_prepopulate_data.cc

```diff
@@ -39,127 +39,135 @@ namespace TemplateURLPrepopulateData {
-         &brave_engines_with_ecosia},
-        {country_codes::CountryCharsToCountryID('B', 'Y'),
-         &brave_engines_with_yandex},
-        {country_codes::CountryCharsToCountryID('C', 'A'),
-         &brave_engines_with_ecosia},
-        {country_codes::CountryCharsToCountryID('C', 'H'),
-         &brave_engines_with_ecosia},
-        {country_codes::CountryCharsToCountryID('D', 'E'), &brave_engines_DE},
-        {country_codes::CountryCharsToCountryID('D', 'K'),
-         &brave_engines_with_ecosia},
-        {country_codes::CountryCharsToCountryID('E', 'S'),
-         &brave_engines_with_ecosia},
-        {country_codes::CountryCharsToCountryID('F', 'I'),
-         &brave_engines_with_ecosia},
-        {country_codes::CountryCharsToCountryID('F', 'R'), &brave_engines_FR},
-        {country_codes::CountryCharsToCountryID('G', 'B'),
-         &brave_engines_with_ecosia},
-        {country_codes::CountryCharsToCountryID('G', 'R'),
-         &brave_engines_with_ecosia},
-        {country_codes::CountryCharsToCountryID('H', 'U'),
-         &brave_engines_with_ecosia},
-        {country_codes::CountryCharsToCountryID('I', 'E'),
-         &brave_engines_AU_IE},
-        {country_codes::CountryCharsToCountryID('I', 'T'),
-         &brave_engines_with_ecosia},
-        {country_codes::CountryCharsToCountryID('K', 'G'),
-         &brave_engines_with_yandex},
-        {country_codes::CountryCharsToCountryID('K', 'R'), &brave_engines_KR},
-        {country_codes::CountryCharsToCountryID('K', 'Z'),
-         &brave_engines_with_yandex},
-        {country_codes::CountryCharsToCountryID('L', 'U'),
-         &brave_engines_with_ecosia},
-        {country_codes::CountryCharsToCountryID('M', 'D'),
-         &brave_engines_with_yandex},
-        {country_codes::CountryCharsToCountryID('N', 'L'),
-         &brave_engines_with_ecosia},
-        {country_codes::CountryCharsToCountryID('N', 'O'),
-         &brave_engines_with_ecosia},
-        {country_codes::CountryCharsToCountryID('N', 'Z'), &brave_engines_NZ},
-        {country_codes::CountryCharsToCountryID('P', 'T'),
-         &brave_engines_with_ecosia},
-        {country_codes::CountryCharsToCountryID('R', 'U'),
-         &brave_engines_with_yandex},
-        {country_codes::CountryCharsToCountryID('S', 'E'),
-         &brave_engines_with_ecosia},
-        {country_codes::CountryCharsToCountryID('T', 'J'),
-         &brave_engines_with_yandex},
-        {country_codes::CountryCharsToCountryID('T', 'M'),
-         &brave_engines_with_yandex},
-        {country_codes::CountryCharsToCountryID('U', 'S'),
-         &brave_engines_with_ecosia},
-        {country_codes::CountryCharsToCountryID('U', 'Z'),
-         &brave_engines_with_yandex}};
+const std::map<int, const std::vector<BravePrepopulatedEngineID>>&
+GetDefaultEnginesByCountryIdMap() {
+  static const std::map<int, const std::vector<BravePrepopulatedEngineID>>
+      default_engines_by_country_id_map = {
```

Comment:
can you try to convert everything in this file to `constexpr` like `fixed_map_set`? see below:
```
static constexpr auto content_v20 =
      base::MakeFixedFlatMap<int, BravePrepopulatedEngineID>({
```

Reply:
Done.

Reply:
All done. Thanks!